src/shader_compiler.cc

```diff
@@ -142,10 +143,13 @@ std::pair<Result, std::vector<uint32_t>> ShaderCompiler::Compile(
     return {Result("Invalid shader format"), results};
   }
 
+  (void)disable_spirv_validation;
```

Comment:
The linter won't like this. Is there some way we can restructure the code so we don't need have a fake access?

Reply:
The checks passed, but perhaps you mean other linters. I think this can be removed once it becomes a member variable.

Reply:
Removed.